src/application/components/Cache/Cache.tsx

```diff
@@ -39,9 +46,11 @@ function filterCache(cache: Cache, searchTerm: string) {
   );
 }
 
+const history = new History("ROOT_QUERY");
```

Comment:
Instantiating this outside of the component allows us to keep the history when changing tabs, otherwise this resets anytime the `Cache` component unmounts.